setup.sh

```diff
@@ -1,25 +1,35 @@
 #!/bin/bash
 
-echo "Instaling Bundler"
-gem install bundler
+# Tells the shell script to exit if it encounters an error
+set -e
+
+# -- Log -----------------------------------------------------------------------
+function msg { echo  "\033[0;37m$1\033[0m"; }
+function msg_ok { echo  "➜\033[1;32m $1 ✔\033[0m"; }
+function msg_run { echo  "➜\033[1;35m $1 $ $2\033[0m"; }
+
+# -- Dependencies --------------------------------------------------------------
+echo "Instaling Bundler!"
+sudo gem install bundler
 bundle install
 
-echo "Instaling or Updating Homebrew"
-which -s brew
-if [[ $? != 0 ]] ; then
-    # Install Homebrew
-    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+echo "Installing Homebrew!"
+if which brew &> /dev/null; then
+	msg_ok "homebrew"
 else
-    brew update
+	msg_run "homebrew" "/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)""
+	/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 fi
 
-echo "Instaling or Updating Swiftlint"
+echo "Instaling/Updating Swiftlint"
 which -s swiftlint
-if [[ $? != 0 ]] ; then
-    brew install swiftlint
-else
+if [[ $? != 1 ]] ; then
+    msg_run "swiftlint" "brew upgrade swiftlint"
```

Comment:
Estou pensando se realmente vale a pena forçar o update do `swiftlint`. Isso talvez possa causar um efeito colateral no sistema da pessoa. Supondo que ela não queira atualizar naquele momento pq determinado update dele o faria ter que trocar diversas coisas num outro projeto X. Se a gente força a atualização através desse script, ela vai ter atualizado sem saber, interferindo talvez no desenvolvimento desse outro projeto X dela. O que acha?